test/test.js

```diff
@@ -22875,7 +22875,7 @@
     QUnit.test('should coerce `n` to an integer', function(assert) {
       assert.expect(1);
 
-      var actual = _.times(2.6, _.indentify);
+      var actual = _.times(2.6, _.identity);
```

Comment:
I looked carefully and found no sign of an `indentify` anywhere (there were other places with this typo, but they've since been removed), there is an `identity` which makes more sense logically.